### ISSUE TITLE
Application window doesn't start minimized

### ISSUE THREAD

**Issue by miquecg** [keepassxreboot/keepassxc#8089]
## Overview
I start KeePassXC automatically on boot and wanted to have its window minimized but for some reason it's always open, so I have to click to hide it.

## Steps to Reproduce
These are the relevant settings:
- [x] Start automatically on boot
- [x] Start minimized
- [x] Minimize window after unlock
- [x] Show icon on system tray
- [x] Hide window to system tray when minimized

## Expected Behavior
Start KeePassXC minimized to system tray.

## Actual Behavior
Application starts with the main window visible.

## Context
KeePassXC - Versión 2.7.1
Revisión: 5916a8f
Qt 5.15.4

Operating System: Arch Linux
Desktop Env: KDE
Plasma desktop: 5.24.5
Plasma framework: 5.94.0
Windowing System: X11

**Comment by miquecg** [comment]
I don't know if it has something to do with this issue the fact that window size it's always reset on every boot.

**Comment by droidmonkey** [comment]
Do you have ~/.cache ram mounted?

**Comment by miquecg** [comment]
Nope.

**Comment by miquecg** [comment]
Now the problem is a bit worse because on some ocasions there are two tray icons after boot. I have the option to only allow one KeePassXC instance checked.

**Comment by droidmonkey** [comment]
This is on your end, you need to cleanup whatever you have set up for auto starting. Are you running multiple varieties of keepassxc? Snap, flatpak, native, appimage? Unfortunately we cannot help you with the information provided. Closing unless there is actionable info we can use.

**Comment by miquecg** [comment]
I did nothing special except for ticking boxes on application settings, so what am I supposed to cleanup? Any pointers would be appreciated.

For the sake of it, I'm just running stock KeePassXC from Arch Linux repos.

**Comment by droidmonkey** [comment]
First run `ps ax | grep keepassxc` to see where the instances are located. Second, check your startup options for your operating systems and make sure there is only on entry for keepassxc there. Last, make sure your operating system isn't set to "start all apps open on last login"

**Comment by miquecg** [comment]
Thanks. I think it was happening because I have set KDE to restore last session. Now I disabled the option on KeePassXC to rely on system behaviour only.

It looks like the check for allowing "only one instance" doesn't prevent this kind of situations. In my case it was totally unintended for instance.

By the way, on three consecutive boots the application window now remains minimized to tray. So probably all boiled down to having conflicting settings with OS.
